### PR TITLE
Use VK_FORMAT_R8G8B8A8_UNORM for DRM ARGB formats

### PR DESCRIPTION
**Pull request by Joshua-Ashton**
DRM formats are not endian-dependent, therefore we should not use the PACK32 formats for this.

### CODE REVIEW

File: src/rendervulkan.cpp [hunk 1]
```diff
@@ -147,8 +147,7 @@ struct {
 	bool bNeedsSwizzle;
 	bool bHasAlpha;
 } s_DRMVKFormatTable[] = {
-	{ DRM_FORMAT_XRGB8888, VK_FORMAT_A8B8G8R8_UNORM_PACK32, true, false },
-	{ DRM_FORMAT_ARGB8888, VK_FORMAT_A8B8G8R8_UNORM_PACK32, true, true },
+	{ DRM_FORMAT_XRGB8888, VK_FORMAT_R8G8B8A8_UNORM, true, true },
```
Comment (emersion) [inline src/rendervulkan.cpp:150]: `bHasAlpha` should probably be set to false here?

Comment (Joshua-Ashton) [reply src/rendervulkan.cpp:150]: typop

Comment (Joshua-Ashton) [reply src/rendervulkan.cpp]: Fixed.

**Comment by emersion** [comment]
Hm, this new commit results in no error and no window when running glxgears in nested mode on an AMD GFX8 card...

**Comment by emersion** [comment]
I got confused, re-tested and works fine now!

**Review by emersion — APPROVED** [review]
LGTM, thanks!

**Pull request merged**